src/utils/metrics.test.ts

```diff
@@ -29,202 +29,277 @@ import { insertOss } from './metrics';
 
 describe('metrics tests', function () {
   describe('insertOss', function () {
-    const defaultPayload: Record<string, any> = {
-      action: 'labeled',
-      sender: {
-        login: 'username',
-        id: 'user_id',
-      },
-      repository: {
-        full_name: 'getsentry/routing-repo',
-        name: 'routing-repo',
-        owner: {
-          type: 'Organization',
+    describe('lableling events', function () {
+      const defaultPayload: Record<string, any> = {
+        action: 'labeled',
+        sender: {
+          login: 'username',
+          id: 'user_id',
         },
-      },
-      organization: {
-        login: 'getsentry',
-      },
-      issue: {
-        number: 1234,
-        created_at: null,
-        updated_at: null,
-        labels: [{ name: 'Product Area: Test' }],
-      },
-      label: {
-        id: 1234,
-        name: 'Status: Test',
-      },
-    };
+        repository: {
+          full_name: 'getsentry/routing-repo',
+          name: 'routing-repo',
+          owner: {
+            type: 'Organization',
+          },
+        },
+        organization: {
+          login: 'getsentry',
+        },
+        issue: {
+          number: 1234,
+          created_at: null,
+          updated_at: null,
+          labels: [{ name: 'Product Area: Test' }],
+        },
+        label: {
+          id: 1234,
+          name: 'Status: Test',
+        },
+      };
 
-    let dateNowSpy;
-    beforeAll(async () => {
-      dateNowSpy = jest
-        .spyOn(Date, 'now')
-        .mockImplementation(() => 1487076708000);
-      await db.migrate.latest();
-      await getLabelsTable().insert({
-        label_name: 'Product Area: Test',
-        channel_id: 'CHNLIDRND1',
-        offices: ['sfo'],
-      });
-      await getLabelsTable().insert({
-        label_name: 'Product Area: Other',
-        channel_id: 'CHNLIDRND1',
-        offices: ['sfo'],
+      let dateNowSpy;
+      beforeAll(async () => {
+        dateNowSpy = jest
+          .spyOn(Date, 'now')
+          .mockImplementation(() => 1487076708000);
```

Comment:
nit: may be a bit more readable if we use moment to generate this timestamp?

Reply:
done